### DISCUSSION TITLE
Enable both SFTP and FTPS, limit user/group to one or the other

### DISCUSSION THREAD

**Discussion by jshute75** [drakkan/sftpgo#1787] [Ideas]
It would be great to be able to limit a user account or group to authenticate through only SFTP or FTPs when both are enabled.

**Comment by chsoft-two** [comment]
You can use the "Denied protocols" setting under "ACLs", either on a user directly or on a group.

**Comment by jshute75** [comment]
Thank you!  I missed this when exploring options.


Jim Shute
Network/System Engineer
Weber Display & Packaging
3500 Richmond Street | Philadelphia, PA 19134
Direct: (215) 644-7939 | Office: (215) 426-3500 x133
We create packaging and displays, design through delivery, that drive customer success.
